src/test/java/step2/StringCalculateTest.java

```diff
@@ -0,0 +1,87 @@
+package step2;
+
+
+import Calculator.StringCalculator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StringCalculateTest {
+    //Calculator calculator;
```

Comment:
각 테스트 코드마다 StringCalculator 인스턴스를 생성하고 계시네요!
테스트 실행시마다 StringCalculator 인스턴스를 생성하도록 중복을 제거하면 좋을 것 같아요!

```suggestion
    private StringCalculator stringCalculator;

    @BeforeEach
    void setUp() {
        stringCalculator = new StringCalculator();
    }
```

아래의 테스트 가이드 링크를 참고하시면 좋을 것 같습니다.
[Junit5 Guide](https://junit.org/junit5/docs/current/user-guide/)